src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java

```diff
@@ -55,7 +89,7 @@ public WebElement forElementVisible(WebElement element) {
     try {
       element.getTagName();
     } catch (WebDriverException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)